### ISSUE TITLE
Guidelines Sub Domain

### ISSUE THREAD

**Issue by rimblas** [insum-labs/plsql-and-sql-coding-guidelines#1]
What sub-domain should we use for the guidelines website?
https://insum-labs.github.io/plsql-and-sql-coding-guidelines/ is cool and all, but long and not memorable.

I would like to see code.insum.ca

Yes, no, other?

**Comment by RichardSoule** [comment]
Standards.insum.ca?

**Comment by rimblas** [comment]
Yes, standards works, but it is harder to type.  Would it have a landing page and let us choose between PL/SQL and APEX standards?

And are they guidelines or standards?

**Comment by RichardSoule** [comment]
I believe the 'official' one should be either:

standards.insum.ca or guidelines.insum.ca

Both are 'harder to type', but are easy to remember.

We can also have re-direct with the others:

code.insum.ca
standards.insum.ca
guidelines.insum.ca

All should work.

**Comment by rimblas** [comment]
I think the redirects will need to be of the form insum.ca/path
Ex: insum.ca/guidelines

Just because I think redirects of subdomains pose other issues but could be wrong.

I'll look into who can do the setup.

**Comment by RichardSoule** [comment]
Redirect are fine.

Anything that is relatively easy to remember.

Maybe we need an insum.ca/shortcuts ;)

**Comment by rimblas** [comment]
Yes! I think we do need an insum.ca/shortcuts

**Comment by neilfernandez** [comment]
insum.ca/guidelines or insum.ca/code sounds great to me!

**Comment by RichardSoule** [comment]
insum.ca/guidelines is now active. Actual sub domain will take a bit longer, so we’ll leave this open until it is in place.
